### ISSUE TITLE
why detach the pred tensor while  communicate the diffptsl2loss

### ISSUE THREAD

**Issue by yjcaimeow** [hht1996ok/ADMap#7]
@hht1996ok  It is very confusing, why you detach the pred (pred_tmp = pred.detach())
when communicate the diffptsl2loss?

Best regards
Yingjie

**Comment by xpyqiubai** [comment]
I have same question about this.
